### PR TITLE
add UX for displaying why a session was excluded

### PR DESCRIPTION
**Pull request by et**
## Summary

This PR takes the tracking of the excluded reason (#5252 and #5283) and shows it on the error instance view.

## How did you test this change?

For each excluded reason:

https://github.com/highlight/highlight/blob/d9f37bd298a7144b38fc7cdd66a2553a5d8c293b/backend/private-graph/graph/schema.graphqls#L17-L23

See the UX:

`IgnoredUser`

Screenshot 2023-05-11 at 2 05 55 PM

`NoError`
Screenshot 2023-05-11 at 1 58 41 PM

`Initializing` or `NoActivity` or `NoUserInteractionEvents`

Screenshot 2023-05-11 at 10 27 52 AM

I chose to overload these since it doesn't make sense to distinguish them to the end user (however, it is useful for us to make the distinction for debugging purposes).

## Are there any deployment considerations?

N/A

### CODE REVIEW

File: frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx [hunk 1]
```diff
@@ -480,80 +481,28 @@ const User: React.FC<{
...
-		return (
-			<Box width="full">
-				<Box pb="20" mt="12">
-					<Text weight="bold" size="large">
-						User details
-					</Text>
-				</Box>
-				<Callout title="We didn't find a session for this error">
-					<Box>
-						<Text size="small" weight="medium" color="moderate">
```
Comment (et) [inline frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx:493]: I cleaned up some of the the duped JSX into `ErrorSessionMissingOrExcluded`. I confirmed it still looks the same as before:

Screenshot 2023-05-11 at 2 07 51 PM

File: frontend/src/pages/ErrorsV2/ErrorInstance/ErrorSessionMissingOrExcluded.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+			target="_blank"
+		>
+			Backend SDK setup
+		</LinkButton>
+	)
+
+	const projectSettings = (
+		<LinkButton
+			kind="secondary"
+			to={`/${projectId}/settings/sessions`}
```
Comment (et) [inline frontend/src/pages/ErrorsV2/ErrorInstance/ErrorSessionMissingOrExcluded.tsx:88]: Per a [slack](https://highlightcorp.slack.com/archives/C0454PWLLLW/p1683828702379919) convo, I updated the settings sessions url from `/recording` to `/sessions` (see the other changes in this PR of what also needed to be updated).

**Comment by et** [comment]
Adding @julian-highlight to review the wording updates.

**Comment by julian-highlight** [comment]
A few questions here. Why does the description of e.g. 'No error' state that there was 'no error for this session'. I feel like that's confusing, as you're on the error group page. Shouldn't it be called 'no session for this error'? Same goes for 'NoActivity'.
@et

**Review by deltaepsilon — APPROVED** [review]

**Comment by et** [comment]
> A few questions here. Why does the description of e.g. 'No error' state that there was 'no error for this session'. I feel like that's confusing, as you're on the error group page. Shouldn't it be called 'no session for this error'? Same goes for 'NoActivity'.

Yeah I hear you. It is a bit confusing. Can you propose an update?

**Review by julian-highlight — APPROVED** [review]

**Pull request merged**